module-2/backend/week_one.md

```diff
@@ -5,30 +5,89 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+ANSWER:
+GET, PUT, POST, PATCH, DELETE
+
 2. What is Sinatra?
+ANSWER:
+It's a framework that understands Ruby and makes easier to build web applications. It allows us to perform different actions and display a different view for each path that we define in the controller. It uses HTTP verbs and a very simplified syntax.
+
 4. What is MVC?
+ANSWER:
+Models/Viewer/Controller is a convention to structure your code when building a web application. The Models is reponsible for creating the Ruby objects (horse.rb), the Viewer takes care of the views that are rendered to the user for each path and the Controller handles the routes and the actions for each one.
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+ANSWER:
+We follow conventions when writing code in general for readability, easy maintenance and keep improving it among all the members of the community. 
+
 6. What types of variables are accessible in our view templates without explicitly passing them?
+ANSWER:
+We can access instance variables that are created in the model (horse.rb) and are declared in the view(index.erb)
+
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
-  
+ANSWER:
+Using the <% %> so we can write Ruby inside this tag in an erb file.

```

Comment:
Double check the code snippet below; it would be in our controller. One of the jobs of the controller is to collect information to use in our views. In order to do that (per question 7 above), we would need to set an **instance variable** here. So the code might look like this:

```ruby
get '/horses/' do
  @count = 1
  erb :index
end
```